util/ibex_config.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+import argparse
+import collections
+import os
+import sys
+
+import hjson
+
+DEFAULT_CONFIG_FILE = 'ibex_configs.hjson'
+
+
+class ConfigException(Exception):
+    pass
+
+
+def verify_config(name, config_dict):
+    """Check config_dict matches expectations:
+        - It's a mapping object e.g. OrderedDict
+        - It's values are either strings or integers"""
```

Comment:
Please have a look at http://google.github.io/styleguide/pyguide.html#383-functions-and-methods for the recommended format doc comments.